ando/tools/generator/tests/test_utils.py

```diff
@@ -0,0 +1,81 @@
+import unittest
+
+from ando.tools.generator.tests.utils import (initialize_test_directory, test_directory)
+from ando.tools.generator.utils import (save_tsv, save_json)
+import pandas as pd
+import os
+from pathlib import Path
+import copy
+
+
+class TestUtils(unittest.TestCase):
+
+    def setUp(self) -> None:
+        initialize_test_directory()
+
+    def test_create_file(self):
+        df = pd.DataFrame()
+        path_to_save = test_directory / "test_create_file.tsv"
+        save_tsv(df, path_to_save)
+        self.assertTrue(Path(path_to_save).exists())
+
+    def test_create_dummy_file(self):
+        df = pd.read_csv(test_directory / 'test_files' / 'participants.tsv', sep='\t')
+        path_to_save = os.path.join(test_directory, "test_create_dummy_file.tsv")
```

Comment:
`test_directory` is already a pathlib object. you can just use `/` for concatentation with the filename

Reply:
fixed